### PR TITLE
Fix image and favicon not show on card link

### PR DESCRIPTION
**Pull request by editcue**
Fix image and favicon not show on card link

### CODE REVIEW

File: src/link_metadata_parser.ts [hunk 1]
```diff
@@ -60,17 +60,60 @@ export class LinkMetadataParser {
...
 
-  private getImage(): string | undefined {
+  private async getImage(): Promise<string | undefined> {
     const ogImage = this.htmlDoc
-      .querySelector("meta[property='og:image']")
-      ?.getAttr("content");
-    if (ogImage) return ogImage;
+        .querySelector("meta[property='og:image']")
+        ?.getAttr("content");
+    console.log(ogImage, `https:${ogImage}`)
```
Comment (nekoshita) [inline src/link_metadata_parser.ts]: please remove debug logging

**Review by nekoshita — COMMENTED** [review]
please run `npm run fix` and make sure `npm run lint` is ok

**Comment by nekoshita** [comment]
Thanks for the PR!!
and sorry for late replay

File: src/link_metadata_parser.ts (hunk 1 above)
Comment (editcue) [reply src/link_metadata_parser.ts]: > please remove debug logging

The debug logging have been removed

**Comment by nekoshita** [comment]
please run `npm run fix` and make sure `npm run lint` is ok!

**Comment by nekoshita** [comment]
Thanks for fixing it!

**Comment by editcue** [comment]
> Thanks for fixing it!

You're welcome.

**Pull request merged**